lib/src/header/Header.tsx

```diff
@@ -120,30 +124,40 @@ const DxcHeader = ({
         {isResponsive && responsiveContent && (
           <MainContainer>
             <ChildContainer padding={padding}>
-              <HamburguerItem tabIndex={tabIndex} underlined={underlined} onClick={handleMenu}>
+              <HamburguerItem tabIndex={tabIndex} onClick={handleMenu}>
                 {hamburgerIcon}
                 <HamburguerTitle>Menu</HamburguerTitle>
               </HamburguerItem>
             </ChildContainer>
-            {
-              <div>
-                <ResponsiveMenu hasVisibility={isMenuVisible}>
-                  <ResponsiveLogoContainer>{headerResponsiveLogo}</ResponsiveLogoContainer>
-                  <BackgroundColorProvider color={colorsTheme.header.menuBackgroundColor}>
-                    <ContentContainerComponent />
-                  </BackgroundColorProvider>
-                  <CloseContainer tabIndex={tabIndex} onClick={handleMenu} className="closeIcon">
-                    {closeIcon}
-                  </CloseContainer>
-                </ResponsiveMenu>
-                <Overlay onClick={handleMenu} hasVisibility={isMenuVisible}></Overlay>
-              </div>
-            }
+            <ResponsiveMenu hasVisibility={isMenuVisible}>
+              <ResponsiveIconsContainer>
+                <ResponsiveLogoContainer>{headerResponsiveLogo}</ResponsiveLogoContainer>
+                <CloseContainer tabIndex={tabIndex} onClick={handleMenu} aria-label="closeIcon">
```

Comment:
`aria-label` value needs to follow the same approach as in the other components:

1. Translation process: we need to add the string value of the aria-label to the variables.js file, in the `const defaultTranslatedComponentLabels`.
2. The value should follow the pattern "Verb noun". For example, the clear action of a text input has `aria-label="Clear field"` (maybe for this case a value "Close menu", what do you think?).

Reply:
Done!